tests/ember_debug/object-inspector-test.js

```diff
@@ -222,21 +222,23 @@ module('Ember Debug - Object Inspector', function(hooks) {
   });
 
   test('Correct mixin order with es6 class', function (assert) {
-
+    // eslint-disable-next-line ember/no-new-mixins
```

Comment:
Should we maybe disable this rule for the whole project or just this whole folder? I think creating mixins in tests in okay.